infrastructure/httpapi/handlers/ibc_channel.go

```diff
@@ -37,17 +37,24 @@ func (handler *IBCChannel) ListChannels(ctx *fasthttp.RequestCtx) {
 		return
 	}
 
-	channelIdOrder := view.ORDER_ASC
+	var listOrder ibc_channel_view.IBCChannelsListOrder
+	view_order_desc := view.ORDER_DESC
+	view_order_asc := view.ORDER_ASC
```

Comment:
Why copying the value here?

Reply:
Because `view.ORDER_DESC` and `view.ORDER_ASC` are constants, they are not addressable.

Here is a reference: https://stackoverflow.com/questions/35146286/find-address-of-constant-in-go

Reply:
Try `primptr`?

Reply:
👍